### PR TITLE
Add PAYPAL_WEB_PROFILE_CONFIG to ecommerce defaults.

### PR DESCRIPTION
**Pull request by jimabramson**
XCOM-319

@rlucioni @clintonb FYI @Nickersoft related to https://github.com/edx/ecommerce/pull/177

I'm still looking for ops reviewers, but you can get started catching stuff that's missing here or just plain wrong.

### CODE REVIEW

File: playbooks/roles/ecommerce/defaults/main.yml [hunk 1]
```diff
@@ -69,6 +69,10 @@ ECOMMERCE_PAYPAL_CLIENT_ID: 'SET-ME-PLEASE'
 ECOMMERCE_PAYPAL_CLIENT_SECRET: 'SET-ME-PLEASE'
 ECOMMERCE_PAYPAL_RECEIPT_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/'
 ECOMMERCE_PAYPAL_CANCEL_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/'
+ECOMMERCE_PAYPAL_WEB_PROFILE_CONFIG:
+    openedx_default:
+      brand_name: 'OpenEdX'
```
Comment (clintonb) [inline playbooks/roles/ecommerce/defaults/main.yml]: We should choose a default value that does not involve edX to avoid potential confusion.

Comment (jimabramson) [inline playbooks/roles/ecommerce/defaults/main.yml]: do you mean for the brand name, or for the profile key?

Comment (clintonb) [inline playbooks/roles/ecommerce/defaults/main.yml]: Brand name.

**Comment by jimabramson** [comment]
@maxrothman please review

**Comment by jimabramson** [comment]
@clintonb fixed brand_name (to match the platform name default)

**Comment by clintonb** [comment]
:+1:

**Comment by rlucioni** [comment]
Your changes in [#177](https://github.com/edx/ecommerce/pull/177) include a "presentation" key in the dictionary. Why is that missing here?

**Comment by jimabramson** [comment]
@rlucioni nice catch, i inadvertently omitted it here.  fixed.

File: playbooks/roles/ecommerce/defaults/main.yml [hunk 2]
```diff
@@ -69,6 +69,11 @@ ECOMMERCE_PAYPAL_CLIENT_ID: 'SET-ME-PLEASE'
 ECOMMERCE_PAYPAL_CLIENT_SECRET: 'SET-ME-PLEASE'
 ECOMMERCE_PAYPAL_RECEIPT_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/'
 ECOMMERCE_PAYPAL_CANCEL_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/'
+ECOMMERCE_PAYPAL_WEB_PROFILE_CONFIG:
+    openedx_default:
```
Comment (rlucioni) [inline playbooks/roles/ecommerce/defaults/main.yml]: In production use, we won't be using an "openedx_default" web profile; we'll have substituted our own brand name and logo image. This is admittedly a minor point, but would you be in favor of changing this key to simply read "default"? It's implied that the defaults provided here are intended for users of Open edX.

**Comment by rlucioni** [comment]
Aside from my proposed simplification of the key name, :+1: .

File: playbooks/roles/ecommerce/defaults/main.yml (hunk 2 above)
Comment (jimabramson) [inline playbooks/roles/ecommerce/defaults/main.yml]: that makes sense, i will change this in both PRs.

**Comment by maxrothman** [comment]
:+1: What should those values be in prod/stage/edge/etc.?

**Comment by rlucioni** [comment]
@jimabramson please record your answer to @maxrothman's question on the [Service Settings wiki page](https://openedx.atlassian.net/wiki/display/XCOM/Service+Settings).

**Comment by jimabramson** [comment]
squashed.   will merge after build.

@maxrothman I updated https://openedx.atlassian.net/wiki/display/XCOM/Service+Settings with the values for prod and stage.  I don't believe we need to support edge at this time.  Note the "logo image" values point to absolute urls at our marketing sites.  Is this OK?

**Comment by jimabramson** [comment]
@maxrothman  I've made changes pursuant to our conversation earlier.  I've also updated with settings wiki linked above.  As of this change, out of the box and with configuration defaults, we have a generic uncustomized openedx branding profile for PayPal checkout pages; in staging and production the image and brand name are overridden for edx.org.  The LMS' hosted assets are used as the image source.

@rlucioni @clintonb FYI.

**Comment by jimabramson** [comment]
abandoning this PR as we won't use configuration to manage this.

**Pull request closed without merging**